src/utils/serverRequests.js

```diff
@@ -0,0 +1,67 @@
+import axios from "axios";
+
+export const getCart = async(dispatch) => {
+  try{
+    const {status, data} = await axios({
+      method: "get",
+      url: "/api/user/cart",
+      headers: {authorization: localStorage.getItem("encodedToken")}
+    });
+    console.log("Calling api get cart");
+    if(status===200){
```

Comment:
remove the console.log

Reply:
Will do it later. I have added it intentionally for now.